installation/scripts/minikube.sh

```diff
@@ -131,6 +131,11 @@ function addDevDomainsToEtcHosts() {
 	log "Domain added to /etc/hosts" green
 }
 
+function increaseFsInotifyMaxUserInstances() {
```

Comment:
can you add a comment in the code why we need to do this?

Reply:
added